CMakeLists.txt

```diff
@@ -213,6 +213,7 @@ if(NOT QTKEYCHAIN_STATIC)
 else()
     add_library(${QTKEYCHAIN_TARGET_NAME} STATIC ${qtkeychain_SOURCES} ${qtkeychain_MOC_OUTFILES} ${qtkeychain_QM_FILES})
 endif()
+set_target_properties( ${QTKEYCHAIN_TARGET_NAME} PROPERTIES DEBUG_POSTFIX "d" )
```

Comment:
I think this is only common on Windows, so this should probably only be applied there

Reply:
New commit, just for WIndows

Reply:
Ejem.. wrong commit message ... should be: Only set suffix in Windows